src/ServiceInsight/App.xaml.cs

```diff
@@ -40,6 +41,7 @@ public partial class App : IAppCommands
     {
         public App()
         {
+            BlobCache.ApplicationName = "ServiceInsight";
```

Comment:
All caching behavior is hit or miss, as in the cache hits, and you get a result, or it misses, and you have to do the extra work to create the value. For breaking changes, new versions, etc etc it just means the cache misses the first time, and then the new schema/value/whatever is stored in the cache.